lib/cli/handle-error.js

```diff
@@ -118,43 +127,68 @@ module.exports = async (exception, options = {}) => {
       ' For debugging logs, run again after setting the',
       ' "SLS_DEBUG=*" environment variable.',
     ].join('');
-    consoleLog(chalk.red(debugInfo));
-    consoleLog(' ');
+    legacy.consoleLog(chalk.red(debugInfo));
+    legacy.consoleLog(' ');
   }
 
   const platform = process.platform;
   const nodeVersion = process.version.replace(/^[v|V]/, '');
-
-  consoleLog(chalk.yellow('  Get Support --------------------------------------------'));
-  consoleLog(`${chalk.yellow('     Docs:          ')}docs.serverless.com`);
-  consoleLog(`${chalk.yellow('     Bugs:          ')}github.com/serverless/serverless/issues`);
-  consoleLog(`${chalk.yellow('     Issues:        ')}forum.serverless.com`);
-
-  consoleLog(' ');
-  consoleLog(chalk.yellow('  Your Environment Information ---------------------------'));
-  consoleLog(chalk.yellow(`     Operating System:          ${platform}`));
-  consoleLog(chalk.yellow(`     Node Version:              ${nodeVersion}`));
-
   const installationModePostfix = await (async () => {
     if (isStandaloneExecutable) return ' (standalone)';
     if (isLocallyInstalled != null) return isLocallyInstalled ? ' (local)' : '';
     return resolveIsLocallyInstalled() ? ' (local)' : '';
   })();
-  consoleLog(
-    chalk.yellow(`     Framework Version:         ${slsVersion}${installationModePostfix}`)
-  );
-  consoleLog(chalk.yellow(`     Plugin Version:            ${sfeVersion}`));
-  consoleLog(chalk.yellow(`     SDK Version:               ${platformClientVersion}`));
-
   const componentsVersion = (() => {
     try {
       return require('@serverless/components/package').version;
     } catch (error) {
       return 'Unavailable';
     }
   })();
-  consoleLog(chalk.yellow(`     Components Version:        ${componentsVersion}`));
-  consoleLog(' ');
+
+  if (command === 'deploy' && serverless) {
```

Comment:
I don't think we should put here any command specific logic.

It's a generic error handler, and it should stay command agnostic as it was

Reply:
Where do you think we should inject such logic then? I was wondering what would be the best place and catch-all error handler seemed like the best place to not miss any errors

Reply:
I take issue is that logic is divided into hooks, and it's hard to do catch all that wraps all involved hooks (?)

And before we simply shown such message on AWS stack update failure, and here we want to cover broader spectrum of deploy (?)

Reply:
Yes, that's correct - previously the message was just printed out during stack failure or no message was printed at all other than generic error message.